crates/torii/graphql/src/tests/models_test.rs

```diff
@@ -198,10 +198,10 @@ mod tests {
 
         // where filter LT and GT
         let records =
-            records_model_query(&schema, "(where: { type_u32GT: 2, type_u64LT: 4 })").await;
+            records_model_query(&schema, "(where: { type_u32GT: 2, type_u16LT: 4 })").await;
         let connection: Connection<Record> = serde_json::from_value(records).unwrap();
         let first_record = connection.edges.first().unwrap();
-        assert_eq!(first_record.node.type_u64, 3);
+        assert_eq!(first_record.node.type_u16, 3);
```

Comment:
Is this expected to be `u16` as `u64` is now treated as a string? Is the `4` then still valid?

Reply:
Yeah, u64 is now a string. 4 is still valid as all uint members in Record share the same value

https://github.com/dojoengine/dojo/blob/main/crates/torii/types-test/src/contracts.cairo#L66

Reply:
Thanks for the context. :+1: